frontend/src/components/product/components/Pagination.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { ButtonBase } from "@material-ui/core";
+
+const Pagination = (props: any) => {
+  const { productPerPage, totalPage, paginate } = props;
```

Comment:
I think we should add parameter `currentPage` that should highlight current page that we are now.